content/en/about/day-in-life-dev.md

```diff
@@ -6,95 +6,111 @@ position: 7
-- The author of the code gets the benefit of different perspectives on code structure and style, as well as having any unexpected bugs / edge cases pointed out.
-- The reviewer gets to learn how other developers are writing code and if there are any useful libraries and techniques that they can adapt for themselves.
-- Last but not least, knowing that others will be looking at the code naturally encourages more effort in keeping quality high.
+- Having multiple sets of eyes on a piece of code is an easy way to evaluate its readability and clarity. It also lowers
+  the bus factor by distributing knowledge of what code lives where and why.
+- The author of the code gets the benefit of different perspectives on code structure and style, as well as having any
+  unexpected bugs / edge cases pointed out.
+- The reviewer gets to learn how other developers are writing code and if there are any useful libraries and techniques
+  that they can adapt for themselves.
+- Last but not least, knowing that others will be looking at the code naturally encourages more effort in keeping
+  quality high.
 
-At Narrative, code reviews happen as a part of the regular development process, i.e. we review all code as it is pushed and there 
-is no specified “weekly code review” meeting where everyone brings snippets for review. We currently use Crucible to perform these reviews, and most reviews are done post-merge. 
+At Narrative, code reviews happen as a part of the regular development process, i.e. relevant developers review all code
+as it is pushed and there is no specified “weekly code review” meeting where everyone brings snippets for review. We
+currently use Github perform these reviews, and most reviews are done post-merge.
 
-However there are occasional caseswhere someone in the team will request a pre-merge review to validate assumptions / have additional 
-pairs of eyes on a critical piece of code.
+However there are occasional cases where someone in the team will request a pre-merge review to validate assumptions /
+have additional pairs of eyes on a critical piece of code.
 
-### Daily standup (2PM EST)
+### Daily standup (14:30 EST)
 
-We meet on Google Meet every day at 1:30 PM EST for a quick, 15-minute conversation to quickly synchronize and address any blockers. 
+We meet on Google Meet every day at 14:30 EST for a quick, 15-minute conversation to quickly synchronize and address
+any blockers.
 
-### Weekly dev meeting
+### Dev Weekly
 
-We gather on Google Meet once a week (Tuesday at 1:30 EST) for a 1-hour discussion with the dev team. This is the opportunity to raise awareness about 
-any issues, suggest improvements to the process, seek feedback on ideas, etc.
+We gather on Google Meet once a week (Tuesday at 14:30 EST) for a 1 hour discussion with the dev team. This is the
+opportunity to raise awareness about any issues, suggest improvements to the process, seek feedback on ideas, etc.
 
 ## Development
 
 ### Code
```

Comment:
Update: removed example projects because they felt a bit stale. Updated the example projects in the job posting, but those feel bigger/aspirational and the ones we have here should be more grounded and day-to-day, but ran out of energy trying to think of what this should be.

If we someone wants to take a crack at this please do 🙇